react/flagMap.tsx

```diff
@@ -0,0 +1,442 @@
+const RCA = React.lazy(() => import('./icons/RCA'))
+const ROU = React.lazy(() => import('./icons/ROU'))
+const RUS = React.lazy(() => import('./icons/RUS'))
+const RWA = React.lazy(() => import('./icons/RWA'))
+const SAU = React.lazy(() => import('./icons/SAU'))
+const SDN = React.lazy(() => import('./icons/SDN'))
+const SEN = React.lazy(() => import('./icons/SEN'))
+const SGP = React.lazy(() => import('./icons/SGP'))
+const SGS = React.lazy(() => import('./icons/SGS'))
+const SLB = React.lazy(() => import('./icons/SLB'))
+const SLE = React.lazy(() => import('./icons/SLE'))
+const SLV = React.lazy(() => import('./icons/SLV'))
+const SMR = React.lazy(() => import('./icons/SMR'))
+const SOM = React.lazy(() => import('./icons/SOM'))
+const SRB = React.lazy(() => import('./icons/SRB'))
+const SSD = React.lazy(() => import('./icons/SSD'))
+const STP = React.lazy(() => import('./icons/STP'))
+const SUR = React.lazy(() => import('./icons/SUR'))
+const SVK = React.lazy(() => import('./icons/SVK'))
+const SVN = React.lazy(() => import('./icons/SVN'))
+const SWE = React.lazy(() => import('./icons/SWE'))
+const SWZ = React.lazy(() => import('./icons/SWZ'))
+const SYC = React.lazy(() => import('./icons/SYC'))
+const SYR = React.lazy(() => import('./icons/SYR'))
+const TCD = React.lazy(() => import('./icons/TCD'))
+const TGO = React.lazy(() => import('./icons/TGO'))
+const THA = React.lazy(() => import('./icons/THA'))
+const TJK = React.lazy(() => import('./icons/TJK'))
+const TKM = React.lazy(() => import('./icons/TKM'))
+const TMP = React.lazy(() => import('./icons/TMP'))
+const TON = React.lazy(() => import('./icons/TON'))
+const TTO = React.lazy(() => import('./icons/TTO'))
+const TUN = React.lazy(() => import('./icons/TUN'))
+const TUR = React.lazy(() => import('./icons/TUR'))
+const TUV = React.lazy(() => import('./icons/TUV'))
+const TWN = React.lazy(() => import('./icons/TWN'))
+const TZA = React.lazy(() => import('./icons/TZA'))
+const UGA = React.lazy(() => import('./icons/UGA'))
+const UKR = React.lazy(() => import('./icons/UKR'))
+const UNK = React.lazy(() => import('./icons/UNK'))
+const URY = React.lazy(() => import('./icons/URY'))
+const USA = React.lazy(() => import('./icons/USA'))
+const UZB = React.lazy(() => import('./icons/UZB'))
+const VCT = React.lazy(() => import('./icons/VCT'))
+const VEN = React.lazy(() => import('./icons/VEN'))
+const VGB = React.lazy(() => import('./icons/VGB'))
+const VIR = React.lazy(() => import('./icons/VIR'))
+const VNM = React.lazy(() => import('./icons/VNM'))
+const VUT = React.lazy(() => import('./icons/VUT'))
+const WSM = React.lazy(() => import('./icons/WSM'))
+const YEM = React.lazy(() => import('./icons/YEM'))
+const ZAF = React.lazy(() => import('./icons/ZAF'))
+const ZMB = React.lazy(() => import('./icons/ZMB'))
+const ZWE = React.lazy(() => import('./icons/ZWE'))
+
+/**
+ * Mapping of country's ISO-3 code to the flag component
+ */
+const flagMap: { [key: string]: React.ComponentType } = {
+  AFG,
```

Comment:
Does it work if you do something like this?
```suggestion
  AFG: React.lazy(() => import('./icons/ZWE'))
```

Having all the country codes twice in the code bothers me a bit.

Reply:
sure! indeed it's better than have it duplicated in the same file

Reply:
done in 1202552